lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2292,6 +2292,67 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      mme_pool_1:
+        type: string
+        format: ipv4
+        example: ''
+      mme_pool_2:
+        type: string
+        format: ipv4
+        example: ''
+      managementServer:
+        $ref: '#/definitions/managementServer'
+      sync_1588:
+        $ref: '#/definitions/sync_1588'
+  sync_1588:
+    description: sync_1588 Configuration for  eNodeb devices.
```

Comment:
```suggestion
    description: sync_1588 Configuration for eNodeb devices.
```

Reply:
Looks like there's a few versions - is there a reference that can be included to a specification number?